src/schema/type/DefinedTerm.graphql

```diff
@@ -0,0 +1,70 @@
+"https://pending.schema.org/DefinedTerm"
+type DefinedTerm implements ThingInterface {
```

Comment:
We had to make DefinedTerm implement ThingInterface, because the value of an annotation can be either a string, or a Thing. This has meant that we need to implement all of the additional fields defined in ThingInterface. I don't like the fact that we have a huge number of definitions here, when really we only need a few (id, creator, additionalType, termCode), but I'm not sure if there is a way of making this definition smaller while still allowing the typing to work by allowing one of these to be used where a field type is Thing.

Reply:
@alastair, it should be possible to use Union types, but I haven't tested that properly with relational fields t.b.h.

I also dislike having unused properties, however, [DefinedTerm](https://schema.org/DefinedTerm) also implements `Thing` on schema.org. So `DefinedTerm` should implement `ThingInterface` accordingly.

Reply:
OK. Will leave the full definition here 👍